modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -48,6 +48,24 @@ def save_auto_claim!(auto_claim, status)
       auto_claim.save!
     end
 
+    def set_evss_response(auto_claim, error)
+      auto_claim.evss_response = []
+      error_messages = get_error_message(error)
+      messages = error_messages[:messages].presence || error_messages
+      messages.uniq.each do |error_message|
+        error_key = get_error_key(error_message)
+        error_text = get_error_text(error_message)
+        auto_claim.evss_response <<
+          { 'key' => error_key,
+            'severity' => 'FATAL',
+            'text' => error_text }
+      end
+
+      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
```

Comment:
@stiehlrod  This all looks really good.  One suggestion I would have is moving this line up to be first thing in the chain once an error is returned.  If something above this has an issue and errors out the claim might not get marked as `errored` I believe.  Maybe like right before the call to `set_evss_response` in `disability_compensation_docker_container_upload.rb` and `modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb`?  Not sure, thoughts?